.github/workflows/nextjs_bundle_analysis.yml

```diff
@@ -0,0 +1,120 @@
+name: "Next.js Bundle Analysis"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - next # change this if your default branch is named differently
```

Comment:
nit: update the comment here.

Reply:
These comments are provided by the template: https://github.com/hashicorp/nextjs-bundle-analysis/blob/main/template.yml.
Let's keep them for easier configuration in the future.